src/phoenix/config.py

```diff
@@ -47,7 +47,10 @@
 Note that if you plan on using SQLite, it's advised to to use a persistent volume
 and simply point the PHOENIX_WORKING_DIR to that volume.
 """
-
+ENV_PHOENIX_SQL_DATABASE_SCHEMA = "PHOENIX_SQL_DATABASE_SCHEMA"
+"""
+The schema to use for SQL databases. This is ignored for SQLite.
```

Comment:
can you link this page or something analogous? Otherwise schema might be a tad "ambiguous" https://www.postgresql.org/docs/current/ddl-schemas.html

Reply:
added